modules/actions/pool/sync-swaps.ts

```diff
@@ -54,6 +54,8 @@ export async function syncSwaps(
     // Enrich with USD values
     const dbEntries = await swapsUsd(dbSwaps, chain);
 
+    console.log(dbEntries);
```

Comment:
guess this will be removed :)

Reply:
add TODO to create batchSwaps from swaps

Reply:
it's needed in the v2 UI, not sure if the v3 UI also uses it. Will try to find out